lua/tardis/controls/sh_control_engine_release.lua

```diff
@@ -10,6 +10,11 @@ TARDIS:AddControl({
                 TARDIS:ErrorMessage(ply, "Controls.EngineRelease.FailedDemat")
             end
         end)
+        self:EngineReleaseVortexArtron(function(result)
```

Comment:
You can spam this control and gain tons of artron energy quickly, should this be possible to do?

Reply:
you'll kill your TARDIS rather soon like that
but I might add a small delay

Reply:
I've done a lot of testing
With the current parameters and default health amount, you kill your TARDIS in 4-6 clicks of this one
So this can not be abused

Reply:
Okay